file-formats/intm/type/zif_aff_intm_v1.intf.abap

```diff
@@ -0,0 +1,111 @@
+INTERFACE zif_aff_intm_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Intelligent Scenario Model</p>
+    "! Intelligent scenario model
+    BEGIN OF ty_intelligent_model,
+      "! <p class="shorttext">Model Name</p>
+      "! Model name
+      name                 TYPE c LENGTH 30,
+      "! <p class="shorttext">Intelligent Scenario</p>
+      "! Intelligent scenario of the model
+      intelligent_scenario TYPE c LENGTH 20,
+      "! <p class="shorttext">Modelling Context ID</p>
+      "! Modelling context ID which is the parent of the model
+      modelling_context_id TYPE c LENGTH 36,
+      "! <p class="shorttext">Model Type</p>
+      "! Model type
+      model_type           TYPE c LENGTH 256,
+      "! <p class="shorttext">Adapter ID</p>
+      "! Adapter ID
+      adapter_id           TYPE c LENGTH 32,
+      "! <p class="shorttext">Native Model Type</p>
+      "! Native model type
+      native_model_type    TYPE c LENGTH 256,
+      "! <p class="shorttext">Model Content</p>
+      "! Model content
+      model_specification  TYPE string,
```

Comment:
is this the model content or the model_specification? It's better when description and field name use the same wording